web/packages/selfhosted/js/ruffle.js

```diff
@@ -22,7 +22,8 @@ try {
 
         currentScriptURL = new URL(".", src);
     }
-} catch (e) {
+    // eslint-disable-next-line no-unused-vars
+} catch (_e) {
```

Comment:
Huh, is this lint disable comment needed even with the underscore prefix?

Reply:
That only works with typescript :(

Reply:
Oh, shoot, I see. Alright.